src/its_preselector/web_relay.py

```diff
@@ -29,6 +29,15 @@ def get_digital_input_value(input) -> bool:
         :return: The boolean value read from the digital input.
         """
 
+    @abstractmethod
+    def get_analog_input_value(input) -> float:
+        """
+        Read the value from an analog input on the web relay.
+
+        :param input: The name or ID of the analog input
```

Comment:
I think this should just be the ID (integer) and not the name (like "door_sensor")

Reply:
corrected.